apps/server/src/app.ts

```diff
@@ -202,13 +204,17 @@ export const startIntegrations = async (config?: { osc: OSCSettings }) => {
 export const shutdown = async (exitCode = 0) => {
   console.log(`Ontime shutting down with code ${exitCode}`);
 
+  //TODO: clear the restore file if it was a normal exit
+  if (exitCode != 0) {
+    restoreService.clear();
+  }
+
```

Comment:
Few questions here:
- Can you please clarify the task behind this todo? 
- Would it make more sense to use the exit code we have signified in app (99), since other exit codes might have other meanings?
- is there a reason not to use triple equals here?